src/tenant/forms.py

```
@@ -55,3 +59,89 @@ class TenantForm(TenantBaseForm):
+        Ensure the email field has a valid value during form cleaning.
+
+        Returns the initial verified email if the field is disabled,
+        otherwise returns the cleaned data from user input.
+
+        Returns:
+            str: The email address for the tenant owner.
+        """
+        if self.fields["email"].disabled:
+            email = (self.verified_data or {}).get("email") or self.initial.get("email")
+            if not email:
+                raise forms.ValidationError("Veridied e-mail address is required.")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)